returnn/datasets/meta.py

```diff
@@ -208,9 +211,11 @@ def __init__(self,
     self.data_map = data_map
     self.dataset_keys = set([m[0] for m in self.data_map.values()])  # type: typing.Set[str]
     self.data_keys = set(self.data_map.keys())  # type: typing.Set[str]
-    assert "data" in self.data_keys
-    self.target_list = sorted(self.data_keys - {"data"})
-    self.default_dataset_key = seq_order_control_dataset or self.data_map["data"][0]
+    self.target_list = sorted(self.data_keys)
```

Comment:
Now this includes any potential input (e.g. "data") as well. Which implies  `available_for_inference=False`. But this would be wrong.

Why do you change the behavior?

If you want to make this more generic, then make it more generic. But don't just change the one specific behavior to some other specific behavior.